File: 2501/250112/2688.cpp
#include <iostream>
#include <vector>

using namespace std;

typedef unsigned long long ull;

vector<ull> ans;

ull arr[66][10];

void calc() {
    for (int i = 0; i < 10; i++) {
        arr[1][i] = 1;
    }
    
    for (int row = 2; row <= 65; row++) {
        arr[row][0] = 1;
        
        for (int col = 1; col < 10; col++) {
            arr[row][col] = arr[row][col - 1] + arr[row - 1][col];
        }
    }
}

int main() {
    calc();
    
    int t, n;
    cin >> t;
    
    for (int i = 0; i < t; i++) {
        cin >> n;
        
        ans.push_back(arr[n + 1][9]);
    }
    
    for (int i = 0; i < t; i++) {
        cout << ans[i] << endl;
    }
    
}

File: 2501/250112/2688.py
arr = []

row0 = [0] * 10
row1 = [1] * 10

arr.append(row0)
arr.append(row1)

for row in range(2, 66):
    line = [1]

    for col in range(1, 10):
        line.append(line[col - 1] + arr[row - 1][col])
    arr.append(line)

t = int(input())

ans = []

for i in range(0, t):
    x = int(input())
    ans.append(arr[x + 1][9])

for i in range(0, t):
    print(ans[i])
